internal/server/telemetry.go

```diff
@@ -147,13 +154,19 @@ func (t *Telemetry) User(id string, name string) {
 	}
 }
 
-func (t *Telemetry) Org(id, userID, name string) {
+func (t *Telemetry) Org(id, userID, name, domain string) {
 	if t == nil {
 		return
 	}
 	err := t.Enqueue(analytics.Group{
-		GroupId:   id,
-		Traits:    analytics.NewTraits().SetName(name),
+		GroupId: id,
+		UserId:  userID,
+		Traits: map[string]interface{}{
+			"name":   name,
+			"$name":  name,
```

Comment:
What is the difference between `name` and `$name` here?

Reply:
One is a "super property". Some tools require both